

std_msgs/Header header # frame_id y stamp

int8[]    ids     # track id's
float32[] x_a     # track positions: x
float32[] y_a     # track positions: y
float32[] theta_a # track headings
float32[] vel_a   # track speeds
